docs/source/example_input/boosted_frame_script.py

```diff
@@ -175,21 +182,21 @@ def dens_func( z, r ):
            zf=zfoc, gamma_boost=boost.gamma0 )
 
     # Convert parameter to boosted frame
-    v_window, = boost.velocity( [ v_window ] )
+    v_window_boosted, = boost.velocity( [ v_window ] )
     # Configure the moving window
-    sim.set_moving_window( v=v_window )
+    sim.set_moving_window( v=v_window_boosted )
 
     # Add a field diagnostic
     sim.diags = [ FieldDiagnostic(diag_period, sim.fld, sim.comm ),
                   ParticleDiagnostic(diag_period,
                     {"electrons":sim.ptcl[0], "bunch":sim.ptcl[2]}, sim.comm),
-                  BoostedFieldDiagnostic( zmin, zmax, c, dt_snapshot_lab,
+                  BoostedFieldDiagnostic( zmin, zmax, v_window, dt_snapshot_lab,
                     Ntot_snapshot_lab, boost.gamma0, fieldtypes=['rho','E','B'],
-                    period=diag_period, fldobject=sim.fld, comm=sim.comm),
-                  BoostedParticleDiagnostic( zmin, zmax, c, dt_snapshot_lab,
-                    Ntot_snapshot_lab, boost.gamma0, diag_period, sim.fld,
-                    select={'uz':[0.,None]}, species={'electrons':sim.ptcl[2]},
-                    comm=sim.comm )
+                    period=record_period, fldobject=sim.fld, comm=sim.comm),
+                  BoostedParticleDiagnostic( zmin, zmax, v_window,
+                    dt_snapshot_lab, Ntot_snapshot_lab, boost.gamma0,
+                    record_period, sim.fld, select={'uz':[0.,None]},
+                    species={'electrons':sim.ptcl[2]}, comm=sim.comm )
                     ]
 
     # Number of iterations to perform
```

Comment:
Given the suggested changes above, I think we need to add a +1 to the iterations to guarantee that the last output is always written, i.e.:

```
# Number of iterations to perform
N_step = int(T_interact/sim.dt)+1
```